src/components/Icon/svgs/LoungeAccessIcon.tsx

```diff
@@ -30,31 +29,37 @@ function LoungeAccessIcon(props) {
             y="0px"
             viewBox="0 0 40 40"
             style={{
+                // @ts-expect-error - react-native-web property
                 enableBackground: 'new 0 0 40 40',
             }}
             xmlSpace="preserve"
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...props}
+            fill={fill}
+            hovered={hovered}
+            pressed={pressed}
         >
             <G>
                 <Path
+                    // @ts-expect-error - react-native-web property
                     className="st0"
                     d="M11,24v-2c0-1.1,0.9-2,2-2s2,0.9,2,2c0,0.6,0.4,1,1,1h8c0.6,0,1-0.4,1-1c0-1.1,0.9-2,2-2s2,0.9,2,2v2 c0,0.4-0.1,0.8-0.3,1.1C28.1,26,27,26.9,27,28h-2v-2H15v2h-2c0-1.1-1.1-2-1.7-2.9C11.1,24.8,11,24.4,11,24z"
                 />
             </G>
             <G>
                 <Path
-                    fill={props.hovered === 'true' || props.pressed === 'true' ? props.fill : themeColors.starDefaultBG}
+                    fill={hovered === 'true' || pressed === 'true' ? fill : themeColors.starDefaultBG}
+                    // @ts-expect-error - react-native-web property
                     className="st1"
                     d="M31,9.8c-0.1-0.2-0.2-0.4-0.5-0.4h-2.1l-0.8-2C27.4,7,27.1,7,27,7c-0.1,0-0.4,0-0.6,0.4l-0.8,1.9h-2.1 c-0.4,0-0.5,0.4-0.5,0.4c0,0.1-0.1,0.4,0.1,0.6l1.6,1.8l-0.6,1.9c-0.1,0.3,0.1,0.5,0.2,0.7c0.1,0,0.3,0.2,0.7,0.1l2-1.1l2,1.2 c0.3,0.2,0.6,0,0.7-0.1c0.1-0.1,0.3-0.3,0.2-0.7l-0.6-2l1.5-1.7C31,10.3,31,10,31,9.8z"
                 />
                 <Polygon
+                    // @ts-expect-error - react-native-web property
                     className="st1"
                     points="28.5,7 28.5,7 28.5,7  "
                 />
             </G>
             <G>
                 <Path
+                    // @ts-expect-error - react-native-web property
```

Comment:
Can you try to augment the `react-native-svg` type? `src/types/modules/`

Reply:
Great idea! I pushed a solution, let me know what you think 😄